packages/shared/src/lib/featureManagement.ts

```diff
@@ -45,11 +44,7 @@ const feature = {
     ExperienceLevelExperiment.Control,
   ),
   featureTheme: new Feature('feature_theme', {}),
-  feedListLayout: new Feature(
-    'feed_list_layout',
-    FeedListLayoutExperiment.Control,
-  ),
-  customFeeds: new Feature('custom_feeds', CustomFeedsExperiment.Control),
+  customFeeds: new Feature('custom_feeds', CustomFeedsExperiment.V1),
```

Comment:
Was it intended to switch the values here too?

Reply:
Oh no thanks for noting, it was testing purposed 🙈